README.md
netrunner-settings-thunderbird
=============================
Containing the thunderbird profile with settings


debian/changelog
default-settings-thunderbird (1.0.15) utopic; urgency=low

  * Updated for TB 31

 -- Leszek Lesner <leszek@zevenos.com>  Wed, 29 Oct 2014 10:39:03 +0100


etc/skel/.thunderbird/so8mthku.default/extensions/{e2fda1a4-762b-4020-b5ad-a41df1933103}/defaults/preferences/lightning.js
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

// This file contains all of the default preference values for Lightning

// Turns on basic calendar logging.
pref("calendar.debug.log", false);
// Turns on verbose calendar logging.
pref("calendar.debug.log.verbose", false);

// addon description
pref("extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.description",
     "chrome://lightning/locale/lightning.properties");
pref("extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.name",
     "chrome://lightning/locale/lightning.properties");
pref("extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.creator",
     "chrome://lightning/locale/lightning.properties");

// general settings
pref("calendar.date.format", 0);
pref("calendar.event.defaultlength", 60);
pref("calendar.task.defaultstart", "none");
pref("calendar.task.defaultstartoffset", 0);
pref("calendar.task.defaultstartoffsetunits", "minutes");
pref("calendar.task.defaultdue", "none");
pref("calendar.task.defaultdueoffset", 60);
pref("calendar.task.defaultdueoffsetunits", "minutes");

// default transparency (free-busy status) of standard and all-day events
pref("calendar.events.defaultTransparency.allday.transparent", true);
pref("calendar.events.defaultTransparency.standard.transparent", false);

// number of days in "Soon" section
pref("calendar.agendaListbox.soondays", 5);

// alarm settings
pref("calendar.alarms.show", true);
pref("calendar.alarms.showmissed", true);
pref("calendar.alarms.playsound", true);
pref("calendar.alarms.soundURL", "chrome://calendar/content/sound.wav");
pref("calendar.alarms.defaultsnoozelength", 5);
pref("calendar.alarms.indicator.show", true);
pref("calendar.alarms.indicator.totaltime", 3600);

// default alarm settings for new event
pref("calendar.alarms.onforevents", 0);
pref("calendar.alarms.eventalarmlen", 15);
pref("calendar.alarms.eventalarmunit", "minutes");

// default alarm settings for new task
pref("calendar.alarms.onfortodos", 0);
pref("calendar.alarms.todoalarmlen", 15);
pref("calendar.alarms.todoalarmunit", "minutes");

// open invitations autorefresh settings
pref("calendar.invitations.autorefresh.enabled", true);
pref("calendar.invitations.autorefresh.timeout", 3);

// iTIP compatibility send mode
// 0 -- Outlook 2003 and following with text/plain and application/ics (default)
// 1 -- all Outlook, but no text/plain nor application/ics
// We may extend the compat mode if necessary.
pref("calendar.itip.compatSendMode", 0);

// whether "notify" is checked by default when creating new events/todos with attendees
pref("calendar.itip.notify", true);

// whether the organizer propagates replies of attendees to all attendees
pref("calendar.itip.notify-replies", false);

// whether email invitation updates are send out to all attendees if (only) adding a new attendee
pref("calendar.itip.updateInvitationForNewAttendeesOnly", false);

//whether changes in email invitation updates should be displayed
pref("calendar.itip.displayInvitationChanges", true);

//whether for delegated invitations a delegatee's replies will be send also to delegator(s)
pref("calendar.itip.notifyDelegatorOnReply", true);

// whether CalDAV (experimental) scheduling is enabled or not.
pref("calendar.caldav.sched.enabled", false);

// 0=Sunday, 1=Monday, 2=Tuesday, etc.  One day we might want to move this to
// a locale specific file.
pref("calendar.week.start", 0);
pref("calendar.weeks.inview", 4);
pref("calendar.previousweeks.inview", 0);

// Show week number in minimonth and multiweek/month views
pref("calendar.view-minimonth.showWeekNumber", true);

// Default days off
pref("calendar.week.d0sundaysoff", true);
pref("calendar.week.d1mondaysoff", false);
pref("calendar.week.d2tuesdaysoff", false);
pref("calendar.week.d3wednesdaysoff", false);
pref("calendar.week.d4thursdaysoff", false);
pref("calendar.week.d5fridaysoff", false);
pref("calendar.week.d6saturdaysoff", true);

// start and end work hour for day and week views
pref("calendar.view.daystarthour", 8);
pref("calendar.view.dayendhour", 17);

// number of visible hours for day and week views
pref("calendar.view.visiblehours", 9);

// time indicator update interval in minutes (0 = no indicator)
pref("calendar.view.timeIndicatorInterval", 15);

// If true, mouse scrolling via shift+wheel will be enabled
pref("calendar.view.mousescroll", true);

// Do not set this!  If it's not there, then we guess the system timezone
//pref("calendar.timezone.local", "");

// Recent timezone list
pref("calendar.timezone.recent", "[]");

// categories settings
// XXX One day we might want to move this to a locale specific file
//     and include a list of locale specific default categories
pref("calendar.categories.names", "");

// Make sure mouse wheel shift and no key actions to scroll lines.
pref("mousewheel.withnokey.action", 0);
pref("mousewheel.withshiftkey.action", 0);

// Disable use of worker threads. Restart needed.
pref("calendar.threading.disabled", false);

// The maximum time in microseconds that a cal.forEach event can take (soft limit).
pref("calendar.threading.latency ", 250);

// Enable support for multiple realms on one server with the payoff that you
// will get multiple password dialogs (one for each calendar)
pref("calendar.network.multirealm", false);

// Set up user agent
pref("calendar.useragent.extra", "Lightning/4.7.1");

// Disable use of system colors in minimonth and calendar views
pref("calendar.view.useSystemColors", false);

// Maximum number of iterations allowed when searching for the next matching
// occurrence of a repeating item in calFilter
pref("calendar.filter.maxiterations", 50);

// Backend to use. false: libical, true: ical.js
//@line 154 "/build/thunderbird-kde/src/thunderbird-45.1.0/calendar/lightning/content/lightning.js"
pref("calendar.icaljs", false);
//@line 156 "/build/thunderbird-kde/src/thunderbird-45.1.0/calendar/lightning/content/lightning.js"

// Calendar integration notification
pref("calendar.integration.notify", true);


etc/skel/.thunderbird/so8mthku.default/extensions/{e2fda1a4-762b-4020-b5ad-a41df1933103}/chrome/calendar/content/calendar/calApplicationUtils.js
/* -*- Mode: javascript; tab-width: 20; indent-tabs-mode: nil; c-basic-offset: 4 -*-
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

Components.utils.import("resource://gre/modules/Services.jsm");

function openAboutDialog()
{
  const SUNBIRD_ID = "{718e30fb-e89b-41dd-9da7-e25a45638b28}";
  var url = (Services.appinfo.ID == SUNBIRD_ID) ?
    "chrome://sunbird/content/aboutDialog.xul" :
    "chrome://messenger/content/aboutDialog.xul" ;
//@line 19 "/builds/slave/tb-rel-c-beta-lx_bld-000000000/build/calendar/base/src/calApplicationUtils.js"
  var features = "chrome,centerscreen,dependent,dialog=no";
//@line 21 "/builds/slave/tb-rel-c-beta-lx_bld-000000000/build/calendar/base/src/calApplicationUtils.js"
  window.openDialog(url, "About", features);
}

/**
 * Opens the release notes page for this version of the application.
 */
function openReleaseNotes()
{
  const SUNBIRD_ID = "{718e30fb-e89b-41dd-9da7-e25a45638b28}";
  if (Services.appinfo.ID == SUNBIRD_ID) {
    var bundle = Services.strings.createBundle("chrome://branding/locale/brand.properties");
    var relNotesURL = bundle.formatStringFromName("releaseNotesURL",[Services.appinfo.version],1)
    launchBrowser(relNotesURL);
  } else {
    openFormattedRegionURL('app.releaseNotesURL');
  }
}

/**
 * Opens region specific web pages for the application like the release notes, the help site, etc. 
 *   aResourceName --> the string resource ID in region.properties to load. 
 */
function openRegionURL(aResourceName)
{
  try {
    var regionBundle = Services.strings.createBundle("chrome://messenger-region/locale/region.properties");
    // the release notes are special and need to be formatted with the app version
    var urlToOpen;
    if (aResourceName == "releaseNotesURL")
      urlToOpen = regionBundle.formatStringFromName(aResourceName, [Services.appinfo.version], 1);
    else
      urlToOpen = regionBundle.GetStringFromName(aResourceName);
      
    var protocolSvc = Components.classes["@mozilla.org/uriloader/external-protocol-service;1"]
                      .getService(Components.interfaces.nsIExternalProtocolService);
    protocolSvc.loadUrl(Services.io.newURI(urlToOpen, null, null));
  } catch (ex) {}
}

/**
 *  Fetches the url for the passed in pref name, formats it and then loads it in the default
 *  browser.
 *
 *  @param aPrefName - name of the pref that holds the url we want to format and open
 */
function openFormattedRegionURL(aPrefName)
{
  var formattedUrl = getFormattedRegionURL(aPrefName);
  
  var protocolSvc = Components.classes["@mozilla.org/uriloader/external-protocol-service;1"].
                               getService(Components.interfaces.nsIExternalProtocolService);
  protocolSvc.loadUrl(Services.io.newURI(formattedUrl, null, null));
}

/**
 *  Fetches the url for the passed in pref name and uses the URL formatter service to 
 *    process it.
 *
 *  @param aPrefName - name of the pref that holds the url we want to format and open
 *  @returns the formatted url string
 */
function getFormattedRegionURL(aPrefName)
{
  return Services.urlFormatter.formatURLPref(aPrefName);
}

/**
 * Launch the given url (string) in the external browser. If an event is passed,
 * then this is only done on left click and the event propagation is stopped.
 *
 * @param url       The URL to open, as a string
 * @param event     (optional) The event that caused the URL to open
 */
function launchBrowser(url, event)
{
  // Bail out if there is no url set, or an event was passed without left-click
  if (!url || (event && event.button != 0)) {
    return;
  }

  // 0. Prevent people from trying to launch URLs such as javascript:foo();
  //    by only allowing URLs starting with http or https.
  // XXX: We likely will want to do this using nsIURLs in the future to
  //      prevent sneaky nasty escaping issues, but this is fine for now.
  if (url.indexOf("http") != 0) {
    Components.utils.reportError ("launchBrowser: " +
                                  "Invalid URL provided: " + url +
                                  " Only http:// and https:// URLs are valid.");
    return;
  }

  Components.classes["@mozilla.org/uriloader/external-protocol-service;1"]
            .getService(Components.interfaces.nsIExternalProtocolService)
            .loadUrl(Services.io.newURI(url, null, null));

  // Make sure that any default click handlers don't do anything, we have taken
  // care of all processing
  if (event) {
      event.stopPropagation();
      event.preventDefault();
  }
}


etc/skel/.thunderbird/so8mthku.default/prefs.js
# Mozilla User Preferences

/* Do not edit this file.
 *
 * If you make changes to this file while the application is running,
 * the changes will be overwritten when the application exits.
 *
 * To make a manual change to preferences, you can visit the URL about:config
 */

user_pref("app.update.lastUpdateTime.addon-background-update-timer", 1443471345);
user_pref("app.update.lastUpdateTime.blocklist-background-update-timer", 1364213027);
user_pref("app.update.lastUpdateTime.search-engine-update-timer", 1443471225);
user_pref("app.update.lastUpdateTime.xpi-signature-verification", 0);
user_pref("browser.cache.disk.capacity", 358400);
user_pref("browser.cache.disk.filesystem_reported", 1);
user_pref("browser.cache.disk.smart_size.first_run", false);
user_pref("browser.cache.disk.smart_size.use_old_max", false);
user_pref("browser.cache.disk.smart_size_cached_value", 358400);
user_pref("browser.search.countryCode", "US");
user_pref("browser.search.region", "US");
user_pref("calendar.categories.names", "Anniversary,Birthday,Business,Calls,Clients,Competition,Customer,Favorites,Follow up,Gifts,Holidays,Ideas,Issues,Meeting,Miscellaneous,Personal,Projects,Public Holiday,Status,Suppliers,Travel,Vacation");
user_pref("calendar.category.color.anniversary", "#333333");
user_pref("calendar.category.color.birthday", "#996633");
user_pref("calendar.category.color.business", "#FFFFCC");
user_pref("calendar.category.color.calls", "#99FF99");
user_pref("calendar.category.color.clients", "#FF9900");
user_pref("calendar.category.color.competition", "#CC9933");
user_pref("calendar.category.color.customer", "#CCCCCC");
user_pref("calendar.category.color.favorites", "#3366FF");
user_pref("calendar.category.color.follow_up", "#6600CC");
user_pref("calendar.category.color.gifts", "#FF99FF");
user_pref("calendar.category.color.holidays", "#663366");
user_pref("calendar.category.color.ideas", "#003333");
user_pref("calendar.category.color.issues", "#99FFFF");
user_pref("calendar.category.color.meeting", "#FFFF66");
user_pref("calendar.category.color.miscellaneous", "#FFCC00");
user_pref("calendar.category.color.personal", "#003333");
user_pref("calendar.category.color.projects", "#FFCC99");
user_pref("calendar.category.color.public_holiday", "#FFCC66");
user_pref("calendar.category.color.status", "#FFFF33");
user_pref("calendar.category.color.suppliers", "#000099");
user_pref("calendar.category.color.travel", "#663300");
user_pref("calendar.category.color.vacation", "#660000");
user_pref("calendar.integration.notify", false);
user_pref("calendar.list.sortOrder", "0f0d9c4f-8bb5-4b96-8522-6b3be4c1a5c5");
user_pref("calendar.preferences.lightning.selectedTabIndex", 0);
user_pref("calendar.registry.0f0d9c4f-8bb5-4b96-8522-6b3be4c1a5c5.calendar-main-default", true);
user_pref("calendar.registry.0f0d9c4f-8bb5-4b96-8522-6b3be4c1a5c5.calendar-main-in-composite", true);
user_pref("calendar.registry.0f0d9c4f-8bb5-4b96-8522-6b3be4c1a5c5.name", "Home");
user_pref("calendar.registry.0f0d9c4f-8bb5-4b96-8522-6b3be4c1a5c5.type", "storage");
user_pref("calendar.registry.0f0d9c4f-8bb5-4b96-8522-6b3be4c1a5c5.uri", "moz-storage-calendar://");
user_pref("calendar.timezone.local", "UTC");
user_pref("calendar.ui.version", 1);
user_pref("datareporting.sessions.current.activeTicks", 1);
user_pref("datareporting.sessions.current.clean", true);
user_pref("datareporting.sessions.current.firstPaint", 3116);
user_pref("datareporting.sessions.current.main", 31);
user_pref("datareporting.sessions.current.startTime", "1464086344383");
user_pref("datareporting.sessions.current.totalTime", 8);
user_pref("datareporting.sessions.currentIndex", 9);
user_pref("datareporting.sessions.previous.0", "{\"s\":1462965845678,\"a\":5,\"t\":46,\"c\":true,\"m\":1058,\"fp\":9634,\"sr\":-1}");
user_pref("datareporting.sessions.previous.1", "{\"s\":1462965901085,\"a\":1,\"t\":15,\"c\":true,\"m\":65,\"fp\":5492,\"sr\":-1}");
user_pref("datareporting.sessions.previous.2", "{\"s\":1464085551987,\"a\":4,\"t\":30,\"c\":true,\"m\":31,\"fp\":2079,\"sr\":-1}");
user_pref("datareporting.sessions.previous.3", "{\"s\":1464085584340,\"a\":5,\"t\":28,\"c\":true,\"m\":61,\"fp\":4921,\"sr\":-1}");
user_pref("datareporting.sessions.previous.4", "{\"s\":1464085614369,\"a\":1,\"t\":7,\"c\":true,\"m\":30,\"fp\":3318,\"sr\":-1}");
user_pref("datareporting.sessions.previous.5", "{\"s\":1464085626213,\"a\":1,\"t\":7,\"c\":true,\"m\":30,\"fp\":3276,\"sr\":-1}");
user_pref("datareporting.sessions.previous.6", "{\"s\":1464085715366,\"a\":4,\"t\":22,\"c\":true,\"m\":40,\"fp\":3351,\"sr\":-1}");
user_pref("datareporting.sessions.previous.7", "{\"s\":1464086309250,\"a\":4,\"t\":23,\"c\":true,\"m\":231,\"fp\":4217,\"sr\":-1}");
user_pref("datareporting.sessions.previous.8", "{\"s\":1464086333634,\"a\":1,\"t\":9,\"c\":true,\"m\":40,\"fp\":4073,\"sr\":-1}");
user_pref("extensions.blocklist.pingCountTotal", 4);
user_pref("extensions.blocklist.pingCountVersion", -1);
user_pref("extensions.bootstrappedAddons", "{\"{f69e22c7-bc50-414a-9269-0f5c344cd94c}\":{\"version\":\"43.4\",\"type\":\"extension\",\"descriptor\":\"/home/netrunner/.thunderbird/so8mthku.default/extensions/{f69e22c7-bc50-414a-9269-0f5c344cd94c}.xpi\",\"multiprocessCompatible\":false,\"runInSafeMode\":false},\"langpack-en-US@thunderbird.mozilla.org\":{\"version\":\"45.1.0\",\"type\":\"locale\",\"descriptor\":\"/usr/lib/thunderbird/extensions/langpack-en-US@thunderbird.mozilla.org.xpi\",\"multiprocessCompatible\":false,\"runInSafeMode\":false}}");
user_pref("extensions.databaseSchema", 17);
user_pref("extensions.e10sBlockedByAddons", true);
user_pref("extensions.enabledAddons", "%7Ba62ef8ec-5fdc-40c2-873c-223b8a6925cc%7D:2.7,%7Be2fda1a4-762b-4020-b5ad-a41df1933103%7D:4.7.1,%7B35f30c40-35d4-11d9-8dbc-000c6e787ef9%7D:3.45.8");
user_pref("extensions.getAddons.cache.lastUpdate", 1464085555);
user_pref("extensions.getAddons.databaseSchema", 5);
user_pref("extensions.installedDistroAddon.{e2fda1a4-762b-4020-b5ad-a41df1933103}", true);
user_pref("extensions.lastAppVersion", "45.1.0");
user_pref("extensions.lastPlatformVersion", "45.1.0");
user_pref("extensions.pendingOperations", false);
user_pref("extensions.systemAddonSet", "{\"schema\":1,\"addons\":{}}");
user_pref("extensions.themefontsizechanger.currentfontsize", "12");
user_pref("extensions.ui.dictionary.hidden", true);
user_pref("extensions.ui.lastCategory", "addons://list/extension");
user_pref("extensions.ui.locale.hidden", false);
user_pref("extensions.xpiState", "{\"app-profile\":{\"elementary-thunderbird@alxlit.addons.mozilla.org\":{\"d\":\"/home/netrunner/.thunderbird/so8mthku.default/extensions/elementary-thunderbird@alxlit.addons.mozilla.org.xpi\",\"e\":false,\"v\":\"0.3.3\",\"st\":1464085137000},\"{a62ef8ec-5fdc-40c2-873c-223b8a6925cc}\":{\"d\":\"/home/netrunner/.thunderbird/so8mthku.default/extensions/{a62ef8ec-5fdc-40c2-873c-223b8a6925cc}\",\"e\":true,\"v\":\"2.7\",\"st\":1464085137000,\"mt\":1464085137000},\"{e2fda1a4-762b-4020-b5ad-a41df1933103}\":{\"d\":\"/home/netrunner/.thunderbird/so8mthku.default/extensions/{e2fda1a4-762b-4020-b5ad-a41df1933103}\",\"e\":true,\"v\":\"4.7.1\",\"st\":1464085553000,\"mt\":1464085553000},\"riseofthetools@rsjtdrjgfuzkfg.com\":{\"d\":\"/home/netrunner/.thunderbird/so8mthku.default/extensions/riseofthetools@rsjtdrjgfuzkfg.com.xpi\",\"e\":false,\"v\":\"1.1.0.0\",\"st\":1464085137000},\"{35f30c40-35d4-11d9-8dbc-000c6e787ef9}\":{\"d\":\"/home/netrunner/.thunderbird/so8mthku.default/extensions/{35f30c40-35d4-11d9-8dbc-000c6e787ef9}.xpi\",\"e\":true,\"v\":\"3.45.8\",\"st\":1464085137000},\"{f69e22c7-bc50-414a-9269-0f5c344cd94c}\":{\"d\":\"/home/netrunner/.thunderbird/so8mthku.default/extensions/{f69e22c7-bc50-414a-9269-0f5c344cd94c}.xpi\",\"e\":true,\"v\":\"43.4\",\"st\":1464085137000}},\"app-global\":{\"langpack-en-US@thunderbird.mozilla.org\":{\"d\":\"/usr/lib/thunderbird/extensions/langpack-en-US@thunderbird.mozilla.org.xpi\",\"e\":true,\"v\":\"45.1.0\",\"st\":1463094532000},\"{972ce4c6-7e08-4474-a285-3208198ce6fd}\":{\"d\":\"/usr/lib/thunderbird/extensions/{972ce4c6-7e08-4474-a285-3208198ce6fd}.xpi\",\"e\":false,\"v\":\"45.1.0\",\"st\":1463329106000}}}");
user_pref("font.name.monospace.x-western", "Noto Sans");
user_pref("font.size.fixed.x-western", 14);
user_pref("general.skins.selectedSkin", "ms-office-jb_v3");
user_pref("gfx.blacklist.suggested-driver-version", "<Anything with EXT_texture_from_pixmap support>");
user_pref("idle.lastDailyNotification", 1364213064);
user_pref("layout.css.devPixelsPerPx", "1.2");
user_pref("ldap_2.servers.eds.description", "EDS Address Book Bootstrapper");
user_pref("ldap_2.servers.eds.dirType", 3);
user_pref("ldap_2.servers.eds.filename", "eds.mab");
user_pref("ldap_2.servers.eds.position", 1);
user_pref("ldap_2.servers.eds.uri", "moz-abedsdirectory://");
user_pref("mail.ab_remote_content.migrated", 1);
user_pref("mail.append_preconfig_smtpservers.version", 2);
user_pref("mail.attachment.store.version", 1);
user_pref("mail.default_charsets.migrated", 1);
user_pref("mail.folder.views.version", 1);
user_pref("mail.openMessageBehavior.version", 1);
user_pref("mail.preferences.advanced.selectedTabIndex", 4);
user_pref("mail.preferences.applications.selectedTabIndex", 0);
user_pref("mail.preferences.display.selectedTabIndex", 0);
user_pref("mail.rights.version", 1);
user_pref("mail.spam.version", 1);
user_pref("mail.ui-rdf.version", 11);
user_pref("mailnews.database.global.datastore.id", "18ed416b-cb65-4fdc-bfed-01d5eebb089");
user_pref("mailnews.quotingPrefs.version", 1);
user_pref("mailnews.start_page.enabled", false);
user_pref("mailnews.start_page_override.mstone", "45.1.0");
user_pref("mailnews.tags.$label1.color", "#FF0000");
user_pref("mailnews.tags.$label1.tag", "Important");
user_pref("mailnews.tags.$label2.color", "#FF9900");
user_pref("mailnews.tags.$label2.tag", "Work");
user_pref("mailnews.tags.$label3.color", "#009900");
user_pref("mailnews.tags.$label3.tag", "Personal");
user_pref("mailnews.tags.$label4.color", "#3333FF");
user_pref("mailnews.tags.$label4.tag", "To Do");
user_pref("mailnews.tags.$label5.color", "#993399");
user_pref("mailnews.tags.$label5.tag", "Later");
user_pref("mailnews.tags.version", 2);
user_pref("network.cookie.prefsMigrated", true);
user_pref("network.predictor.cleaned-up", true);
user_pref("places.database.lastMaintenance", 1364213064);
user_pref("places.history.expiration.transient_current_max_pages", 100410);
user_pref("plugin.importedState", true);
user_pref("plugin.state.java", 2);
user_pref("plugin.state.libnpgoogletalk", 2);
user_pref("plugin.state.libnpo1d", 2);
user_pref("privacy.sanitize.timeSpan", 0);
user_pref("signon.importedFromSqlite", true);
user_pref("storage.vacuum.last.index", 0);
user_pref("storage.vacuum.last.places.sqlite", 1364213064);
user_pref("toolkit.telemetry.cachedClientID", "c5f28b33-1b9b-42da-ad4e-556a94153c33");
user_pref("toolkit.telemetry.previousBuildID", "20160515162628");
user_pref("toolkit.telemetry.prompted", 2);
user_pref("xpinstall.whitelist.add", "");
user_pref("xpinstall.whitelist.add.36", "");
